### PR TITLE
Update integrations.yaml with APImetrics

### PR DESCRIPTION
**Pull request by SophieMcK**

### CODE REVIEW

File: data/ecosystem/integrations.yaml [hunk 1]
```diff
@@ -310,3 +310,9 @@
   components: [C++, Java, Go, Python]
   oss: true
   cncf: incubating
+- name: APImetrics
+  url: https://apicontext.com/
+  docsUrl: https://docs.apimetrics.io/docs/export-with-opentelemetry
+  components: [Exporter]
```
Comment (svrnm) [inline data/ecosystem/integrations.yaml:316]: would it be possible to know how those exporters are implemented? so are you using opentelemetry collectors, or is this implemented with one of the language SDKs?

Comment (neil-gall) [reply data/ecosystem/integrations.yaml:316]: They're implemented using the Python SDK.

Comment (svrnm) [reply data/ecosystem/integrations.yaml:316]:
```suggestion
  components: [Python]
```

We need to do better in the future on how we distinguish these integrations, but let's have it like that for now.

**Review by svrnm — APPROVED** [review]

**Comment by SophieMcK** [comment]
Is there any chance that this could be listed as APIContext and not APImetrics

**Comment by svrnm** [comment]
> Is there any chance that this could be listed as APIContext and not APImetrics

You can provide a PR to update the file. Note that we have changed the way how we store integrations, you will find them in data/registry now

**Pull request merged**